web/src/lib/types/types.ts

```diff
@@ -146,6 +146,17 @@ export type ProfileFields =
       keyof typeof profileFieldsFromConfig)
   | keyof BusinessUser;
 
+export type ProfileLabelField = Exclude<ProfileFields, keyof BusinessUser | "businessPersona">;
```

Comment:
can we name this type something else? We already have a React component named `FieldLabelProfile` and this is very similar.